core/src/main/java/com/ibm/wala/core/util/scope/JUnitEntryPoints.java

```diff
@@ -132,8 +132,7 @@ public static Iterable<Entrypoint> make(IClassHierarchy cha) {
   private static boolean isTestEntryPoint(TypeName typeName) {
     // WALA uses $ to refers to inner classes. We have to replace "$" by "."
     // to make it a valid class name in Java source code.
-    String javaName =
-        StringStuff.jvmToReadableType(typeName.getPackage() + "." + typeName.getClassName());
+    String javaName = StringStuff.jvmToReadableType(typeName.toString().replace('$', '.'));
```

Comment:
It would be good to have an inner class test as well.

Reply:
@khatchad can you sketch out a test case that would let us test this?  I'm not familiar with how nested classes are used with JUnit 4 / 5

Reply:
I can have a look at this soon.

Reply:
I think we just need to make sure we are getting the entry point corresponding to a method declared in a nested type.

Reply:
https://github.com/wala/WALA/pull/1409